### PR TITLE
Add Infof

### PR DESCRIPTION
**Pull request by guilhem**
follow action toolkit "info"
https://github.com/actions/toolkit/blob/main/packages/core/src/core.ts#L168

### CODE REVIEW

File: actions.go [hunk 1]
```diff
@@ -272,6 +272,13 @@ func (c *Action) Fatalf(msg string, args ...interface{}) {
 	os.Exit(1)
 }
 
+// Infof prints a info-level message. The arguments follow the standard Printf
+// arguments.
+func (c *Action) Infof(msg string, args ...interface{}) {
+	// ::info <c.fields>::<msg, args>
+	fmt.Fprintln(c.w, msg, args)
```
Comment (sethvargo) [inline actions.go:279]: This should be `c.IssueCommand` like the other calls.

**Review by sethvargo — CHANGES_REQUESTED** [review]

File: actions.go (hunk 1 above)
Comment (guilhem) [reply actions.go:279]: I did it at first, but after looking to SDK, it's not using issue for "info".
But I have to update comment

Comment (sethvargo) [reply actions.go:279]: I was under the impression that the "old" way was deprecated

Comment (guilhem) [reply actions.go:279]: I really have no idea.
All examples use standard output

**Pull request merged**